service/controller/clusterapi/v29/cluster_resource_set.go

```diff
@@ -60,48 +50,6 @@ const (
 	minAllocatedSubnetMaskBits = 16
 )
 
-type ClusterResourceSetConfig struct {
```

Comment:
I tried to consolidate the encrypter creation and moved the config magic into their own files.